### ISSUE TITLE
Edone fails to launch

### ISSUE THREAD

**Issue by samis** [DaveMDS/edone#4]
When launching edone, I get the following error:

```
Traceback (most recent call last):
  File "/usr/bin/edone", line 6, in <module>
    sys.exit(main())
  File "/usr/lib/python3.4/site-packages/edone/main.py", line 43, in main
    win = EdoneWin()
  File "/usr/lib/python3.4/site-packages/edone/gui.py", line 117, in __init__
    b.content = Icon(hbox1, standard='add')
  File "efl/elementary/icon.pyx", line 227, in efl.elementary.icon.Icon.__init__ (efl/elementary/icon.c:2084)
  File "efl.evas_object.pxi", line 209, in efl.evas.Object._set_properties_from_keyword_args (efl/evas/efl.evas.c:51432)
  File "efl.eo.pyx", line 272, in efl.eo.Eo._set_properties_from_keyword_args (efl/eo/efl.eo.c:3275)
  File "efl/elementary/icon.pyx", line 298, in efl.elementary.icon.Icon.standard.__set__ (efl/elementary/icon.c:2681)
RuntimeWarning: Setting standard icon failed
```

The installed version of python-efl reported by my package manager is 1.14.

**Comment by DaveMDS** [comment]
Hi, the problem is that your icon theme do not provide the "add" icon.
I fixed (workaround) this in the last commit. it is possible that it will fail on other icons for you,
please reopen the issue if this is the case.

thanks for reporting

**Comment by samis** [comment]
It does indeed fail on other icons (right now it's the 'find' icon.) However, considering I'm using the default theme of EFL / Enlightenment, it may be worth properly fixing it somehow.

**Comment by NuLogicSystems** [comment]
This Issue persists:

> $ edone
> Traceback (most recent call last):
>   File "/usr/bin/edone", line 6, in <module>
>     sys.exit(main())
>   File "/usr/lib/python3.5/site-packages/edone/main.py", line 43, in main
>     win = EdoneWin()
>   File "/usr/lib/python3.5/site-packages/edone/gui.py", line 136, in **init**
>     en.content_set('end', Icon(en, standard='find', size_hint_min=(20,20)))
>   File "efl/elementary/icon.pxi", line 38, in efl.elementary.**init**.Icon.**init** (efl/elementary/**init**.c:154226)
>   File "efl/evas/efl.evas_object.pxi", line 209, in efl.evas.Object._set_properties_from_keyword_args (efl/evas/efl.evas.c:51771)
>   File "efl/eo/efl.eo.pyx", line 273, in efl.eo.Eo._set_properties_from_keyword_args (efl/eo/efl.eo.c:3311)
>   File "efl/elementary/icon.pxi", line 109, in efl.elementary.__init__.Icon.standard.**set** (efl/elementary/**init**.c:154823)
> RuntimeWarning: Setting standard icon failed

**Comment by DaveMDS** [comment]
This should now be fixed by commit 1333af6b8616f197f4c7216ab1c2d8e3140ed409

Please note that the EFL theme start providing a full icon set from the 1.18 release, so you will probably fail to see some icons. BUT now the app should not fail miserable, you will just see a missing icon.

Please let me know if this fix the issue for you.

Thanks for reporting

**Comment by NuLogicSystems** [comment]
Yes, that fixed the issue.
Thanks Dave.
